src/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -148,7 +148,7 @@ public string ClipboardString
             }
         }
 
-        private string _title;
+        private readonly string _title;
```

Comment:
Please do not make these readonly. But you doing so made me notice some other problems though, these should be modified somewhere. The readonly on the _joystickStates property is fine though

Reply:
readonly keyword removed